test/colony-token-integrations.js

```diff
@@ -0,0 +1,81 @@
+/* globals artifacts */
+import chai from "chai";
+import bnChai from "bn-chai";
+
+const { expect } = chai;
+chai.use(bnChai(web3.utils.BN));
+
+const EtherRouter = artifacts.require("EtherRouter");
+const IColony = artifacts.require("IColony");
+const IColonyNetwork = artifacts.require("IColonyNetwork");
+const ERC20Mintable = artifacts.require("ERC20Mintable");
+
+contract("Colony Token Integration", () => {
+  let colony;
+  let erc20Mintable;
+  let colonyNetwork;
+
+  before(async () => {
+    const etherRouter = await EtherRouter.deployed();
+    colonyNetwork = await IColonyNetwork.at(etherRouter.address);
+  });
+
+  beforeEach(async () => {
+    // Instantiate an openzeppelin ERC20Mintable token instance
+    erc20Mintable = await ERC20Mintable.new();
+    const { logs } = await colonyNetwork.createColony(erc20Mintable.address);
+    const { colonyAddress } = logs[0].args;
+    colony = await IColony.at(colonyAddress);
+    await colony.setRewardInverse(100);
+  });
+
+  describe("when working with openzeppelin-solidity/ERC20Mintable token", () => {
+    it("should be able to correctly claim tokens in the colony funding pots", async () => {
+      await erc20Mintable.mint(colony.address, 100);
+
+      let colonyRewardPotBalance = await colony.getPotBalance(0, erc20Mintable.address);
+      let colonyPotBalance = await colony.getPotBalance(1, erc20Mintable.address);
+      let colonyTokenBalance = await erc20Mintable.balanceOf(colony.address);
+      expect(colonyRewardPotBalance).to.be.zero;
+      expect(colonyPotBalance).to.be.zero;
+      expect(colonyTokenBalance).to.be.eq.BN(100);
+
+      await colony.claimColonyFunds(erc20Mintable.address);
+
+      colonyRewardPotBalance = await colony.getPotBalance(0, erc20Mintable.address);
+      colonyPotBalance = await colony.getPotBalance(1, erc20Mintable.address);
+      colonyTokenBalance = await erc20Mintable.balanceOf(colony.address);
+      expect(colonyRewardPotBalance).to.be.eq.BN(1);
+      expect(colonyPotBalance).to.be.eq.BN(99);
+      expect(colonyTokenBalance).to.be.eq.BN(100);
+    });
+  });
+
+  describe("when working with openzeppelin-solidity/ERC20Mintable token owned by the colony", () => {
+    beforeEach(async () => {
+      await erc20Mintable.addMinter(colony.address);
+      await erc20Mintable.renounceMinter();
```

Comment:
Looks like these two lines are a no-op?

Reply:
Why? https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/access/roles/MinterRole.sol#L30

Reply:
Ah, it's adding the colony.address and renouncing the coinbase account. Nevermind.

Reply:
It's one shade more implicit that I'd like but not sure a simple alternative so #yolo.